build.boot

```diff
@@ -161,7 +158,8 @@
 (defn- seed-simulator-templates
   "Seeds the DB with simulator templates"
   [conn]
-  (->> (load-file "resources/datomic/simulator-templates.edn")
+  (->> (read-all "resources/datomic/simulator-templates.dtm")
+       (first)
```

Comment:
Datomic transactions require nested vectors. In `seed-simulator-templates` we want to operate on each datom before transacting it so we need to map them. For some reason the vectors are nested when using `io/resource` differently (an extra layer). This one is special cased because it's the only one that needs to further operate on the data. If you have an idea for improving it let me know. I agree there is a bit of a code smell here.